proto/internalapi/scanner/v4/vulnerability_report.proto

```diff
@@ -0,0 +1,66 @@
+// This contains protobuf types in pair with ClairCore's types, with
+// minimal differences. See https://github.com/quay/claircore for comments
+// on the fields.
+
+syntax = "proto3";
+
+package v4;
+
+import "google/protobuf/timestamp.proto";
+import "internalapi/scanner/v4/common.proto";
+
+option go_package = "v4";
+
+message VulnerabilityReport {
+    message Vulnerability {
+        enum Severity {
+            UNKNOWN     = 0;
+            NEGLIGIBLE  = 1;
+            LOW         = 2;
+            MEDIUM      = 3;
+            HIGH        = 4;
+            CRITICAL    = 5;
+        }
+
+        enum ArchOp {
+            OP_INVALID        = 0;
+            OP_EQUALS         = 1;
+            OP_NOT_EQUALS     = 2;
+            OP_PATTERN_MATCH  = 3;
+        }
+
+        message VersionRange {
+            NormalizedVersion lower = 1;
+            NormalizedVersion upper = 2;
+        }
+
+        string  id                       = 1;
+        string  updater                  = 2;
+        string  name                     = 3;
+        string  description              = 4;
+        google.protobuf.Timestamp issued = 5;
+        string  links                    = 6;
+        Severity severity                = 7;
+        Severity normalized_severity     = 8;
```

Comment:
We should have our StackRox/ACS normalization here, which is Low, Moderate, Important, Critical (Red Hat's severities)

Reply:
5180a749e9..91adad1bf2